src/components/Imprint/Imprint.tsx

```diff
@@ -0,0 +1,42 @@
+import {DefaultTypography as Typography} from "@common/components";
```

Comment:
Sonst doppelter import
```suggestion
import { DefaultTypography as Typography, DefaultLink as Link } from '@common/components'
```